packages/react-native/Libraries/Network/RCTNetworkTask.h

```diff
@@ -35,13 +35,12 @@ typedef NS_ENUM(NSInteger, RCTNetworkTaskStatus) {
 @property (nonatomic, copy) RCTURLRequestResponseBlock responseBlock;
 @property (nonatomic, copy) RCTURLRequestProgressBlock uploadProgressBlock;
 
-@property (nonatomic, readonly) RCTNetworkTaskStatus status;
-
 - (instancetype)initWithRequest:(NSURLRequest *)request
                         handler:(id<RCTURLRequestHandler>)handler
                   callbackQueue:(dispatch_queue_t)callbackQueue NS_DESIGNATED_INITIALIZER;
 
 - (void)start;
 - (void)cancel;
+- (RCTNetworkTaskStatus)getNetworkTaskStatus;
```

Comment:
This should just be `status`.

> Use “get” only for methods that return objects and values indirectly

https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html

You could also keep this as an `@property` and implement a getter that uses the instance field, so the dot-syntax doesn't need to be changed.